cmd/kubeadm/app/util/etcd/etcd.go

```diff
@@ -187,11 +197,18 @@ func (c *Client) RemoveMember(id uint64) ([]Member, error) {
 	defer cli.Close()
 
 	// Remove an existing member from the cluster
-	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
-	resp, err := cli.MemberRemove(ctx, id)
-	cancel()
+	var lastError error
+	var resp *clientv3.MemberRemoveResponse
+	err = wait.ExponentialBackoff(addRemoveBackoff, func() (bool, error) {
+		resp, err = cli.MemberRemove(context.Background(), id)
+		if err == nil {
+			return true, nil
+		}
+		lastError = err
+		return false, nil
+	})
 	if err != nil {
-		return nil, err
+		return nil, lastError
```

Comment:
Do we need the `lastError` var? Because we are checking `err`, so if `err != nil`  it'll always be equal to `lastError`.